src/pages/Lending/components/LendingHeader.tsx

```diff
@@ -128,6 +132,16 @@ export const LendingHeader = () => {
               </CardFooter>
             </Card>
           </Flex>
+
+          <Alert status='warning' variant='subtle'>
+            <AlertIcon />
+            <AlertDescription>
+              {translate('lending.haltedAlert')}
+              <Link isExternal href='https://discord.gg/n7F4z5Cn' ml={1} color='text.link'>
+                {translate('lending.halterMoreDetails')}
+              </Link>
+            </AlertDescription>
+          </Alert>
```

Comment:
See https://github.com/shapeshift/web/pull/8681#discussion_r1933793406

Reply:
Done in [59f35c1](https://github.com/shapeshift/web/pull/8681/commits/59f35c1b7b628ff50975e1d958516465cee23df4)